package.json

```diff
@@ -53,102 +53,89 @@
     "serve:angular": "nx serve angular",
     "prepack": "yarn build:core && yarn build:all",
     "test": "nx run-many --target=test --all",
-    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main && yarn build:core",
-    "ng": "nx"
+    "postinstall": "ngcc --properties es2020 browser module main"
   },
   "dependencies": {
-    "@angular/animations": "~13.2.0",
-    "@angular/common": "~13.2.0",
-    "@angular/compiler": "~13.2.0",
-    "@angular/core": "~13.2.0",
-    "@angular/forms": "~13.2.0",
-    "@angular/platform-browser": "~13.2.0",
-    "@angular/platform-browser-dynamic": "~13.2.0",
-    "@angular/router": "~13.2.0",
-    "@jscutlery/semver": "2.21.6",
-    "@ledgerhq/hw-transport": "^6.20.0",
-    "@ledgerhq/hw-transport-webhid": "^6.20.0",
-    "@ledgerhq/logs": "^6.10.0",
+    "@angular/animations": "~14.0.0",
+    "@angular/common": "~14.0.0",
+    "@angular/compiler": "~14.0.0",
+    "@angular/core": "~14.0.0",
+    "@angular/forms": "~14.0.0",
+    "@angular/platform-browser": "~14.0.0",
+    "@angular/platform-browser-dynamic": "~14.0.0",
+    "@angular/router": "~14.0.0",
+    "@jscutlery/semver": "^2.25.2",
+    "@ledgerhq/hw-transport": "^6.27.1",
```

Comment:
Upgrades to other packages like this means we'll need to do some sort of regression once we're happy with the upgrade of Nx as it may have inadvertently broken things for our integrations